multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -250,6 +253,9 @@ def __init__(
             self._problem_type = None
             self._pipeline = ZERO_SHOT_IMAGE_CLASSIFICATION
 
+        if problem_type is not None and problem_type.lower() == NER:
```

Comment:
Based on our previous discussions, I think we shall use pipeline for these advanced pipelines, problem_type is for primitive tasks like classification/regression. We could consider this is a later pr.